Sources/CryptoSwift/BlockMode/CFB.swift

```diff
@@ -21,20 +21,29 @@ public struct CFB: BlockMode {
     /// Invalid IV
     case invalidInitializationVector
   }
+    
+  public enum SegmentSize: Int {
+    case cfb8 = 1 // Encrypt byte per byte
+    case cfb128 = 16 // Encrypt 16 bytes per 16 bytes (default)
+  }
 
   public let options: BlockModeOption = [.initializationVectorRequired, .useEncryptToDecrypt]
   private let iv: Array<UInt8>
+  private let segmentSize: SegmentSize
+  public let customBlockSize: Int?
```

Comment:
the block size can be either 16 or 1, if such can you elaborate on why it should accept any block size value?

I see it as a 
```swift
blockSize = SegmentSize.cfb128.rawValue
```

Reply:
Here `customBlockSize` has a getter purpose for `BlockMode`, and is set to `segmentSize.rawValue` in `init` method. (In case it is not nil, it overrides default AES blockSize of 16 bytes)